tools/image-info

```diff
@@ -331,6 +345,10 @@ def append_filesystem(report, tree, *, is_ostree=False):
         if fstab:
             report["fstab"] = fstab
 
+        sysconfig = read_sysconfig(tree)
+        if sysconfig:
```

Comment:
Since your `result` currently always has two keys inside, the condition is always `true`.

Reply:
sysconfig's default will now be an empty dict if no files are found. This condition will then be false.